subsys/bluetooth/shell/iso.c

```diff
@@ -181,11 +181,17 @@ static int cmd_cig_create(const struct shell *sh, size_t argc, char *argv[])
 			return -ENOEXEC;
 		}
 
-		param.interval = interval;
+		param.c_to_p_interval = interval;
+		param.p_to_c_interval = interval;
 	} else {
-		param.interval = 10000;
+		param.c_to_p_interval = 10000;
+		param.p_to_c_interval = 10000;
 	}
-	cis_sdu_interval_us = param.interval;
+	/* cis_sdu_internal_us is used to increase the sequence number
+	 * setting it to the lowest interval number ensures that it
+	 * is increased at least once per interval
+	 */
+	cis_sdu_interval_us = MIN(param.c_to_p_interval, param.p_to_c_interval);
```

Comment:
Not sure I follow this one. Shouldn't this be set to one of those depending on the role?
The `cis_sdu_interval_us` is later used to calculate the TX sequence number in 511.

```
	cis_sn_last = get_next_sn(cis_sn_last, &cis_sn_last_updated_ticks,
				  cis_sdu_interval_us);
```

Reply:
Ah, nice catch. Given that it's used to generate the sequence number, it must be `c_to_p_interval` for central, and  `p_to_c_interval` for peripheral